package.json

```
@@ -14,11 +14,12 @@
   ],
   "type": "module",
   "files": [
-    "index.js"
+    "index.js",
+    "launchSearch.js"
   ],
   "scripts": {
     "prepare:husky": "husky install",
-    "prepare:esbuild": "esbuild index.ts --bundle --packages=external --outfile=index.js --platform=node --format=esm --tree-shaking=true",
+    "prepare:esbuild": "node esbuild.js",
```

I see you took my suggestion literally! I did not mean to suggest that you rewrite this command-line one-liner as a JavaScript file. I meant that if you refer to the esbuild.js script that I referred to, you could work out the _corresponding_ command line arguments for building multiple entry points. I think that it is probably just:

```suggestion
    "prepare:esbuild": "esbuild index.ts launchSearch.ts --bundle --packages=external --outdir=. --platform=node --format=esm --tree-shaking=true",
```